sdk/resourcemanager/azure-resourcemanager/pom.xml

```diff
@@ -270,6 +270,24 @@
           </excludes>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>com.azure.tools</groupId>
+        <artifactId>codesnippet-maven-plugin</artifactId>
+        <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure.tools:codesnippet-maven-plugin;external_dependency} -->
+        <configuration>
+          <additionalReadmes>
+            <additionalReadme>
+              <root>../README.md</root>
+              <glob>**/README.md</glob>
+            </additionalReadme>
+            <additionalReadme>
+              <root>../docs</root>
+              <glob>**/*.md</glob>
+            </additionalReadme>
+          </additionalReadmes>
+        </configuration>
+      </plugin>
```

Comment:
Can we move this to the parent pom and not have to configure it here?

Reply:
This is specific to resourcemanager, same goes for configurations like this in other POMs, so it won't be able to be moved into the parent POM.